From blp@nicira.com Thu Sep 10 23:25:11 2009
Received: from FTLPMAILMX02.citrite.net (10.9.154.224) by
	LONPMAILMX02.citrite.net (10.30.224.163) with Microsoft SMTP Server (TLS)
	id 8.1.393.1; Thu, 10 Sep 2009 23:25:11 +0100
Received: from SMTP02.CITRIX.COM (10.9.154.239) by FTLPMAILMX02.citrite.net
	(10.9.154.224) with Microsoft SMTP Server id 8.1.393.1; Thu, 10 Sep 2009
	18:25:09 -0400
Received: from mail.geoviz.com (HELO leadville.nicira.com)
	([209.172.104.10]) by SMTP02.CITRIX.COM with ESMTP; 10 Sep 2009 18:25:08
	-0400
Received: from hardrock.nicira.com ([172.16.0.20])	by leadville.nicira.com
	with esmtp (Exim 4.67)	(envelope-from <blp@nicira.com>)	id
	1Mls4c-0005vJ-Vk; Thu, 10 Sep 2009 15:25:02 -0700
Received: from blp by hardrock.nicira.com with local (Exim 4.67)
	(envelope-from <blp@hardrock.nicira.com>)	id 1Mls4g-0004Uj-SN; Thu, 10 Sep
	2009 15:25:06 -0700
From: Ben Pfaff <blp@nicira.com>
To: "jpettit@nicira.com" <jpettit@nicira.com>, Ian Campbell <Ian.Campbell@eu.citrix.com>
CC: Ben Pfaff <blp@nicira.com>
Date: Thu, 10 Sep 2009 23:24:51 +0100
Subject: [PATCH 6/6] ovs-vsctl: Style fixes for testsuite.
Thread-Topic: [PATCH 6/6] ovs-vsctl: Style fixes for testsuite.
Thread-Index: AcoyZZAaNiQH+aovTzWSqf/yO5+aoA==
Message-ID: <1252621491-17203-6-git-send-email-blp@nicira.com>
References: <1252621491-17203-1-git-send-email-blp@nicira.com>
In-Reply-To: <1252621491-17203-1-git-send-email-blp@nicira.com>
Accept-Language: en-GB, en-US
Content-Language: en-US
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-Exchange-Organization-AuthSource: FTLPMAILMX02.citrite.net
X-MS-Has-Attach: 
X-MS-TNEF-Correlator: 
x-sbrs: None
x-mesageid: 64739375
x-ironport-server: ftlpip02.citrite.net
x-remote-ip: 209.172.104.10
x-policy: $LOW_THROTTLE
x-ironport-anti-spam-filtered: true
x-ironport-anti-spam-result: AqwFAF8ZqUrRrGgK/2dsb2JhbACBU5pswwOEGAWBVA
x-ironport-av: E=Sophos;i="4.44,366,1249272000";   
	d="scan'208";a="64739375"
Content-Type: text/plain; charset="iso-8859-1"
MIME-Version: 1.0
X-Evolution-Source: imap://ianca@lonpmailmx01.citrite.net/
Content-Transfer-Encoding: 8bit

---
 tests/ovs-vsctl.at |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/tests/ovs-vsctl.at b/tests/ovs-vsctl.at
index 676333c..94f34dd 100644
--- a/tests/ovs-vsctl.at
+++ b/tests/ovs-vsctl.at
@@ -59,8 +59,8 @@ m4_define([CHECK_IFACES],
 AT_SETUP([add-br a])
 AT_KEYWORDS([ovs-vsctl])
 AT_CHECK([RUN_OVS_VSCTL([add-br a])])
-AT_CHECK([cat conf], [0],
-  [bridge.a.port=a
+AT_CHECK([cat conf], [0], [dnl
+bridge.a.port=a
 ])
 CHECK_BRIDGES([a])
 CHECK_PORTS([a])
@@ -70,8 +70,8 @@ AT_CLEANUP
 AT_SETUP([add-br a, add-br b])
 AT_KEYWORDS([ovs-vsctl])
 AT_CHECK([RUN_OVS_VSCTL([add-br a], [add-br b])])
-AT_CHECK([cat conf], [0],
-  [bridge.a.port=a
+AT_CHECK([cat conf], [0], [dnl
+bridge.a.port=a
 bridge.b.port=b
 ])
 CHECK_BRIDGES([a], [b])
@@ -114,8 +114,8 @@ AT_KEYWORDS([ovs-vsctl])
 AT_CHECK([RUN_OVS_VSCTL(
    [add-br a], 
    [add-bond a bond0 a1 a2 a3])])
-AT_CHECK([cat conf], [0],
-  [bonding.bond0.slave=a1
+AT_CHECK([cat conf], [0], [dnl
+bonding.bond0.slave=a1
 bonding.bond0.slave=a2
 bonding.bond0.slave=a3
 bridge.a.port=a
@@ -134,8 +134,8 @@ AT_CHECK([RUN_OVS_VSCTL(
   [add-port a a1],
   [add-port b b1],
   [del-port a a1])])
-AT_CHECK([cat conf], [0],
-  [bridge.a.port=a
+AT_CHECK([cat conf], [0], [dnl
+bridge.a.port=a
 bridge.b.port=b
 bridge.b.port=b1
 ])
@@ -152,8 +152,8 @@ AT_CHECK([RUN_OVS_VSCTL(
   [add-br a], 
   [add-bond a bond0 a1 a2 a3],
   [del-port a bond0])])
-AT_CHECK([cat conf], [0],
-  [bridge.a.port=a
+AT_CHECK([cat conf], [0], [dnl
+bridge.a.port=a
 ])
 CHECK_BRIDGES([a])
 CHECK_PORTS([a])
-- 
1.6.3.3

